lms/djangoapps/shoppingcart/processors/CyberSource.py

```diff
@@ -403,7 +404,7 @@ def get_processor_exception_html(exception):
         '234': dedent(_(
             """
             There is a problem with our CyberSource merchant configuration.  Please let us know at {0}
-            """.format(settings.PAYMENT_SUPPORT_EMAIL))),
+            """).format(settings.PAYMENT_SUPPORT_EMAIL)),
```

Comment:
This would be better without the `dedent` call.
```python
'234': _(
    "There is a problem with our CyberSource merchant configuration.  Please let us know at {0}"
).format(settings.PAYMENT_SUPPORT_EMAIL)
```